public/index.html

```diff
@@ -39,13 +39,17 @@
       </template>
     </div>
     <style type="text/css">
-    #root{height:100%}.loader{display:flex;align-items:center;flex-direction:column;justify-content:center;position:absolute;top:0;right:0;left:0;bottom:0}.loader img{transform:scale(0);animation:load 1s infinite alternate; animation-delay: 1.2s;}@keyframes load{from{transform:scale(0.9)}to{transform:scale(1)}}
+    #root{height:100%}.loader{display:flex;align-items:center;flex-direction:column;justify-content:center;position:absolute;top:0;right:0;left:0;bottom:0}.loader svg{transform:scale(0);animation:load 1s infinite alternate; animation-delay: 1.2s;}@keyframes load{from{transform:scale(0.9)}to{transform:scale(1)}}
     @font-face{font-family:'Proxima Nova';src:url(%PUBLIC_URL%/fonts/ProximaNova-Medium.woff2) format("woff2");font-weight:500;font-style:normal;font-display:swap}
     @font-face{font-family:'Proxima Nova';src:url(%PUBLIC_URL%/fonts/ProximaNova-Regular.woff2) format("woff2");font-weight:400;font-style:normal;font-display:swap}@font-face{font-family:'Proxima Nova';src:url(%PUBLIC_URL%/fonts/ProximaNova-Semibold.woff2) format("woff2");font-weight:600;font-style:normal;font-display:swap}
     </style>
     <div id="root">
       <div class="loader">
-        <img width="128" height="128" src="%PUBLIC_URL%/logo.svg" alt="Sanbase logo" />
+        <svg width="128" height="128" viewBox="0 0 32 32" fill="none">
+          <path fill="#fff" d="M32 16a16 16 0 11-32 0 16 16 0 0132 0z"/>
+          <path fill="#181B2B" d="M8.23 16a1.7 1.7 0 01-1.71 1.69A1.7 1.7 0 014.8 16c0-.93.77-1.69 1.72-1.69.94 0 1.71.76 1.71 1.69zM25.48 17.69A1.7 1.7 0 0027.2 16a1.7 1.7 0 00-1.72-1.69A1.7 1.7 0 0023.77 16c0 .93.77 1.69 1.71 1.69zM19.43 9.61a6.5 6.5 0 00-4.4-.67c-1.38.32-2.57 1.26-2.8 2.84-.18 1.3.22 2.33.9 3.14a9.31 9.31 0 002.27 1.77c.81.5 1.48.9 1.97 1.38.46.44.7.9.7 1.5 0 .69-.2 1.04-.43 1.24-.24.22-.63.38-1.19.42a5.43 5.43 0 01-3.3-.97l-1.21 1.56a7.42 7.42 0 004.65 1.37 3.92 3.92 0 002.4-.94 3.48 3.48 0 001.07-2.68c0-1.26-.55-2.19-1.28-2.9-.7-.67-1.6-1.2-2.32-1.65a7.52 7.52 0 01-1.8-1.36c-.35-.42-.55-.9-.45-1.61.08-.59.5-1.02 1.28-1.2a4.5 4.5 0 013 .5l.94-1.74z"/>
```

Comment:
Colors? (dark mode)

Reply:
Looks pretty for me :) We have the same logo on SAN project page as well
![image](https://user-images.githubusercontent.com/24521041/115427063-c9dc5e80-a209-11eb-9c22-cc63079584a2.png)